microprofile/jwt-auth/src/main/java/io/helidon/microprofile/jwt/auth/JwtAuthProvider.java

```diff
@@ -610,6 +610,13 @@ private InputStream locateStream(String uri) throws IOException {
             InputStream is;
 
             URL url = Thread.currentThread().getContextClassLoader().getResource(uri);
+            if (url == null) {
```

Comment:
Why is this change needed for this PR ?
Should this go in a separate PR ?

Reply:
Because this failed as part of the `module-path` based tests, and I needed to fix it. The build does not pass without it.